### ISSUE TITLE
Arbiter: Add ability to perform hyperparameter search over loss function weights

### ISSUE THREAD

**Issue by AlexDBlack** [eclipse/deeplearning4j#5173]
This isn't currently possible.

Migrated from: https://github.com/deeplearning4j/Arbiter/issues/145
